### ISSUE TITLE
Cannot `npm install` outside of kitware network

### ISSUE THREAD

**Issue by TristanWright** [Kitware/paraviewweb#349]
```
computer:paraviewweb tristan$ npm i
npm WARN deprecated wrench@1.5.8: wrench.js is deprecated! You should check out fs-extra (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for. Thanks for all the usage over the years.
npm WARN deprecated mapbox-gl-function@1.3.0: This package has been merged with the 'mapbox-gl-style-spec' package
npm WARN deprecated mapbox-gl-js-supported@1.1.0: This project has been renamed to mapbox-gl-supported. Install using mapbox-gl-supported instead.
npm WARN deprecated unitbezier@0.0.0: This module has moved: switch to @mapbox/unitbezier
npm WARN deprecated shelf-pack@1.1.0: This module is now under the @mapbox namespace: install @mapbox/shelf-pack instead
npm WARN deprecated feature-filter@2.2.0: This package has been merged into the 'mapbox-gl-style-spec' package
npm WARN deprecated mapbox-gl-style-spec@8.11.0: This package has been replaced by @mapbox/mapbox-gl-style-spec
npm WARN deprecated geojson-area@0.1.0: This module is now under the @mapbox namespace: install @mapbox/geojson-area instead
npm ERR! fetch failed http://www.kitware.com/in-situ-data/npm/tonic-arctic-sample-data-1.2.2.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed http://www.kitware.com/in-situ-data/npm/tonic-arctic-sample-data-1.2.2.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed http://www.kitware.com/in-situ-data/npm/tonic-arctic-sample-data-1.2.2.tgz
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/tristan/.nvm/versions/node/v7.0.0/bin/node" "/Users/tristan/.nvm/versions/node/v7.0.0/bin/npm" "i"
npm ERR! node v7.0.0
npm ERR! npm  v3.10.8

npm ERR! fetch failed with status code 404
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/tristan/Documents/work/paraviewweb/npm-debug.log
```

**Comment by TristanWright** [comment]
Pretty sure 404 comes from this dependency in `package.json`

`"tonic-arctic-sample-data": "http://www.kitware.com/in-situ-data/npm/tonic-arctic-sample-data-1.2.2.tgz",`

**Comment by jourdain** [comment]
It has been fix in ParaViewWeb/master.

**Comment by TristanWright** [comment]
k looks good now, HPCCloud just has an old version.

**Comment by jourdain** [comment]
The test data for ParaViewWeb became unreachable after Kitware updated their website...

**Comment by jourdain** [comment]
```
v2.2.78
@jourdain jourdain released this 14 days ago

2.2.78 (2017-02-27)

Bug Fixes
 - TestData: Fix URL for test data (9b6b3d02)
```
